single_kernel_mongo/managers/cluster.py

```diff
@@ -102,6 +102,9 @@ def share_secret_to_mongos(self, relation: Relation) -> None:
         if int_tls_ca := self.state.tls.get_secret(label_name=SECRET_CA_LABEL, internal=True):
             relation_data[ClusterStateKeys.INT_CA_SECRET.value] = int_tls_ca
 
```

Comment:
please add a small comment with your expert ldap+mongo knowledge on why config-server must share this data to mongos

Reply:
Yes sure !